replica/database.hh

```diff
@@ -1499,6 +1499,13 @@ public:
     // Apply the mutation atomically.
     // Throws timed_out_error when timeout is reached.
     future<> apply(schema_ptr, const frozen_mutation&, tracing::trace_state_ptr tr_state, db::commitlog_force_sync sync, db::timeout_clock::time_point timeout);
+    // Apply mutations atomically.
+    // On restart, either all mutations will be replayed or none of them.
+    // All mutations must belong to the same commitlog domain.
```

Comment:
Can you remind me please what is "commit log domain"?

Reply:
It's `db::commitlog` instance. Before this patch, there was a single domain used by all writes. Now we have two domains, for system_schema and regular.

Reply:
You mean after the next patch?

Reply:
Yes.